doc/api/globals.md

```diff
@@ -598,13 +598,30 @@ The object that acts as the namespace for all W3C
 [WebAssembly][webassembly-org] related functionality. See the
 [Mozilla Developer Network][webassembly-mdn] for usage and compatibility.
 
+## `reportError(error)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+<!-- type=global -->
+
+* `error` {Error|any} an error to report.
+
+The `reportError(error)` method triggers an uncaught exception with the error
+`error`. This will crash the Node.js process by default.
```

Comment:
might be good to link to uncaught exception handling somewhere here so people can take action on the "will crash" bit of the doc.